src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * (radius * radius);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: "
```

Comment:
Avoid using System.out.println in the draw method. Consider using a more flexible logging system or returning the string to be printed by the caller.